storage/elasticsearch/client.go

```diff
@@ -87,7 +108,35 @@ func (c *ElasticSearchClient) request(method string, path string, query string,
 	return req.Do(&response)
 }
 
+func (e *ElasticIndex) increaseEntries() {
+	e.lock.Lock()
+	e.entriesCounter += 1
+	e.lock.Unlock()
+}
+
+func getIndexPath(name string) string {
+	t := time.Now()
+	return fmt.Sprintf("skydive_%s_v%d_%d-%02d-%02d_%02d-%02d-%02d",
+		name, indexVersion, t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second())
+}
+
+func (c *ElasticSearchClient) GetIndexAlias() string {
+	return fmt.Sprintf("%s_%s", indexPrefix, c.index.name)
+}
+
+func (c *ElasticSearchClient) GetIndexAllAlias() string {
+	return fmt.Sprintf("%s_%s", indexPrefix, indexAllAlias)
+}
+
+func (c *ElasticSearchClient) countEntries() int {
+	curEntriesCount, _ := c.connection.Count(c.index.path, "", nil, "")
+	logging.GetLogger().Infof("#####       %s real entries in %s is %d", c.index.name, c.index.path, curEntriesCount.Count)
```

Comment:
Please remove the `#####     ` prefix and I guess this should be a `Debugf` rather